src/validation/aging/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 set(AGING_VALIDATION_DIR ${MAM_X_VALIDATION_DIR}/aging)
-set(AGING_VALIDATION_SCRIPTS_DIR ${MAM_X_VALIDATION_DIR}/scripts)
+set(AGING_VALIDATION_SCRIPTS_DIR ${MAM_X_VALIDATION_DIR}/aging)
```

Comment:
Do you think this is still needed (as opposed to `AGING_VALIDATION_DIR`)?

Reply:
You are correct that is not needed. Removing.